kafka-connect/Dockerfile.deb8

```diff
@@ -48,12 +48,6 @@ RUN echo "===> Installing ${COMPONENT}..." \
     && apt-get update \
     && apt-get install -y apt-transport-https \
     && apt-get install -y procps \
```

Comment:
Should we just be removing this entire image then since nothing is being installed in this docker image now?

Reply:
Not entirely empty, this is still installing *a* connector bellow:

```
confluent-hub install confluentinc/kafka-connect-gcs:latest --no-prompt
```

And Connect team are still reviewing possibilities to re-use this. They may end up deciding to just remove this entirely at that date.

Reply:
totally missed that part below.